website/docs/add-secure-apps/applications/manage_apps.md

```diff
@@ -35,6 +43,49 @@ When multiple policies/groups/users are attached, you can configure the _Policy
 - Require users to pass all bindings/be member of all groups (ALL), or
 - Require users to pass either binding/be member of either group (ANY)
 
+## Application Entitlements
+
+<span class="badge badge--preview">Preview</span>
+<span class="badge badge--version">authentik 2024.12+</span>
+
+Application entitlements can be used through authentik to manage authorization within an application (what areas of the app can users or groups can access). Entitlements are scoped to a single application and can be bound to multiple users/groups (binding policies is not currently supported), giving them access to the entitlement. An application can either check for the name of the entitlement (via the `entitlements` scope), or via attributes stored in entitlements.
+
+An authentik admin can create an entitlement [in the Admin interface](#create-an-application-entitlement) or using the [authentik API](../../developer-docs/api/api.md).
+
+Because entitlements exist within an application, names of entitlements must be unique within an application. This also means that entitlements are deleted when an application is deleted.
+
+### Using entitlements
+
+Entitlements a user has access to can be retrieved using the `user.app_entitlements()` function in property mappings/policies. This function needs to be passed the specific application for which to get the entitlements. For example:
+
+```python
+entitlements = [entitlement.name for entitlement in request.user.app_entitlements(provider.application)]
+return {
+    "entitlements": entitlements,
+}
+```
+
+### Attributes
+
+Each entitlement can store attributes similar to user and group attributes. These attributes can be accessed in property mappings and passed to applications via `user.app_entitlements_attributes`. For example:
+
+```python
+attrs = request.user.app_entitlements(provider.application)
+return {
+    "my_attr": attrs.get("my_attr")
+}
+```
+
+### Create an application entitlement
+
+1. To create an application entitlement open the Admin interface and navigate to **Applications -> Applications**.
```

Comment:
Suggestion: "To create an application entitlement:" might work better as a phrase appearing _above_ the list, since it's the topic of this list of steps, not the topic of Item 1 or whatever.

```
To create an application entitlement:

1. Open the Admin Interface and navigate to **Applications → Applications**...
```